studio/components/interfaces/Organization/BillingSettingsV2/Subscription/PlanUpdateSidePanel.tsx

```diff
@@ -201,10 +201,8 @@ const PlanUpdateSidePanel = () => {
                             <Button
                               block
                               disabled={
-                                // no self-serve downgrades from team plan right now
-                                (plan.id !== PRICING_TIER_PRODUCT_IDS.TEAM &&
-                                  ['enterprise'].includes(subscription?.plan?.id || '')) ||
-                                !canUpdateSubscription
+                                // No self-serve downgrades from Enterprise
+                                subscription?.plan?.id === 'enterprise' || !canUpdateSubscription
```

Comment:
i cant remember but do we have some UI to indicate why they the button is disabled for enterprise downgrade?

we might need to add a tooltip content below for this condition, like how we're doing for !canUpdateSubscription

something like "Please contact support if you'd like to change your plan"